client/src/components/TaskCard/TaskCard.js

```diff
@@ -35,9 +35,15 @@ const TaskCard = ({
   };
 
   const reflectionDoneHandler = async (reflectionComments, satisfaction) => {
-    await createTaskReflectionHandler(id, reflectionComments, satisfaction);
-    await toggleTaskHandler(id);
-    toggleReflectionModal();
+    const success = await createTaskReflectionHandler(
```

Comment:
I would rename `success` to `isTaskReflectionCreated`

Reply:
Renamed.